include/daemons.h

```diff
@@ -31,6 +56,60 @@ typedef struct
     char email    [EMAILSZ];
 }   regmaildb_req;
 
+#define VERIFYDB_VKEY_SIZE (160)
+
+// verifydb status codes.
+#define VERIFYDB_OK (0)
+#define VERIFYDB_ERROR (-1)
+
+// verifydb vmethod keys.
+#define VMETHOD_UNSET (0)
+
+typedef struct {
+    size_t cb;
```

Comment:
This deserves a better name. Based on your usage, the shortest unambiguous name I can think of is `bytecount`.

Reply:
This file uses `cb` across the board. I think we should keep the name for consistency purpose, unless we want to change them all.